pkg/logs/config/integration_config.go

```diff
@@ -98,19 +115,29 @@ func buildLogsAgentIntegrationsConfig(config *viper.Viper, ddconfdPath string) e
 
 	integrationConfigFiles := availableIntegrationConfigs(ddconfdPath)
 	logsSourceConfigs := []*IntegrationConfigLogSource{}
+	integrationsStatus := []status.Integration{}
 
 	for _, file := range integrationConfigFiles {
 		var integrationConfig IntegrationConfig
 		var viperCfg = viper.New()
+
+		integrationName := strings.TrimSuffix(file, filepath.Ext(file))
```

Comment:
is this always true?

Reply:
it's a logic taken from the `agent`'s code:
```
checkName := strings.TrimSuffix(f.Name(), cfgExt)
```

Reply:
ok, could we move this to a method and use this method. This might be updated one day, and we don't want our logic to break because of code duplication